daprdocs/content/en/dotnet-sdk-docs/_index.md

```diff
@@ -57,6 +57,13 @@ Put the Dapr .NET SDK to the test. Walk through the .NET quickstarts and tutoria
       <a href="{{< ref dotnet-actors >}}" class="stretched-link"></a>
     </div>
   </div>
+  <div class="card">
+    <div class="card-body">
+      <h5 class="card-title"><b>Workflow</b></h5>
+      <p class="card-text">Create and manage workflows with state, reminders/timers, and methods in .NET.</p>
```

Comment:
I'm not so sure I understand this description. Was it copy/pasted/modified from another sample? This sample doesn't currently do anything explicitly with reminders/timers or "methods" - though admittedly I'm not sure what _methods_ refers to in this context (maybe something to do with service invocation?).

Reply:
Oh, yep - I had copied from the Actors card. This PR is still a WIP, just now getting back to it.